apps/web-giddh/src/app/ledger/ledger.component.html

```diff
@@ -382,7 +382,7 @@ <h3 class="ledgerHead">
                                                                 <div class="d-flex align-items-center"
                                                                     >
                                                                     <span class="pl-05">
-                                                                        <input type="checkbox"
+                                                                        <input type="checkbox" id="select-entry-cr"
```

Comment:
ID provided is same as the element on line 270 and several other places below this code in the same file. Please make sure the elements have unique IDs